src/components/navigation/www/navigation.jsx

```diff
@@ -77,6 +69,23 @@ class Navigation extends React.Component {
         if (!this.props.user) return;
         return `/users/${this.props.user.username}/`;
     }
+
+    setupMessagePolling (minutes) {
```

Comment:
if it's not too much trouble, it might make more sense to use seconds as the unit -- that seems to be more of a convention than minutes.

Reply:
Good point! This name is left over from a previous iteration. Renamed to pollForMessages since that's actually what it is doing.